core/src/main/java/com/linecorp/armeria/common/stream/EventLoopStreamMessage.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.stream;
+
+import io.netty.channel.EventLoop;
+
+class EventLoopStreamMessage<T> extends AbstractStreamMessageAndWriter<T> {
```

Comment:
`AbstractStreamMessage` already considers multi-threads. How about directly implementing `StreamMessageAndWriter` for efficiency?